public/app/features/alerting/unified/components/rule-editor/ExpressionsEditor.tsx

```diff
@@ -18,7 +18,6 @@ interface Props {
   queries: AlertQuery[];
   onRemoveExpression: (refId: string) => void;
   onUpdateRefId: (oldRefId: string, newRefId: string) => void;
-  onUpdateExpressionType: (refId: string, type: ExpressionQueryType) => void;
```

Comment:
This one looks suspicious. Don't we really use it?

Reply:
We no longer use it, we used to allow switching a expression type but now we force delete + add